buildstockbatch/base.py

```diff
@@ -612,6 +613,31 @@ def validate_resstock_version(project_file):
 
         return True
 
+    @staticmethod
+    def validate_openstudio_version(project_file):
+        """
+        Checks the required version of OpenStudio against the version being used
+        """
+        cfg = get_project_configuration(project_file)
+
+        version_rb = os.path.join(cfg['buildstock_directory'], 'resources/hpxml-measures/HPXMLtoOpenStudio/resources/version.rb')
+        if os.path.exists(version_rb):
+            versions = {}
+            with open(version_rb, 'r') as f:
+                for line in f:
+                    line = line.strip()
+                    for tool in ['OS_Version']:
+                        if line.startswith(tool):
+                            lhs, rhs = line.split('=')
+                            version, _ = rhs.split('#')
+                            versions[tool] = eval(version.strip())
+            OS_Version = versions['OS_Version']
+            if cfg['os_version'] != OS_Version:
```

Comment:
cfg may not have os_version (it's optional), in which case it should use the default version. (Maybe we can specify the default in the yaml schema?)
Also, don't we want inequality comparison? If cfg version is greater than OS_Version, wouldn't it be fine?

Reply:
Actually, no. OS-HPXML requires an exact version, not a minimum version: https://github.com/NREL/OpenStudio-HPXML/blob/master/HPXMLtoOpenStudio/resources/version.rb#L8-L16.

Reply:
Soky, sounds good then. Might still need to handle missing "OS_Version" in the yaml though.

Reply:
Yep, got that in.